manager/dispatcher/discord.go

```diff
@@ -64,6 +64,16 @@ func (d *discord) SetDispatcher(firstRunMsg bool, preStat tp.StateFlag, notifyIn
 		d.entry.Store(pUnique, newEntries)
 		d.reminderCron.Start()
 	} else if reminderState == tp.OFF {
+		count := 0
```

Comment:
using instead of count, 
Can we check the type and skip NOT to do anything? 
such as 
```
entries, _ := d.entry.Load(pUnique)
		if _, ok := entries.([]cron.EntryID); ok {
			for _, entity := range entries.([]cron.EntryID) {
				{
					d.reminderCron.Remove(entity)
				}
				d.reminderCron.Stop()
			}
		}
```

Reply:
Yes, you're right. It's better to check the value exist or not.
I updated it on c03dc8916571cce26ccb69faa2b97c96a69446d6